universal.md

```diff
@@ -77,89 +80,156 @@ The `objects` each correspond to items that the client may wish to process, and
 
   Standard OCI annotations as per existing standards, this is metadata that may have common fields across different objects. Arguably this should be a map of string to string array so that annotation keys can be repeated, so they map better to other structures (such as http headers) that allow repetition.
 
-- **any other data** *unspecified*
+- **`ext_`*** *unspecified*
```

Comment:
I'm still a bit nervous about this one as it's ripe for namespace conflicts for groups that wish to use similarly named properties.
I'm thinking about namespaces scoped sections. Does anyone know of any good examples to use?